src/fixed.ts

```diff
@@ -82,30 +87,36 @@ export default class FixedThreadPool<Data = any, Response = any> {
    * @param data The input for the task specified.
    * @returns Promise that is resolved when the task is done.
    */
-  public async execute (data: Data): Promise<Response> {
+  // eslint-disable-next-line @typescript-eslint/promise-function-async
+  public execute (data: Data): Promise<Response> {
```

Comment:
should async keywork be there ?

Reply:
Not really, we return a **raw** promise here and don't have the `await` keyword in use in the body of this function

There is even a rule for that https://eslint.org/docs/rules/require-await

Reply:
I would prefer to stay with async keyword since that was previously present

Reply:
Oh... now it wants a `return await res`, but we don't want to await that here...
And this is coming from `ts-standard` and this is not really up-to-date :confused:

Reply:
So to explain what happens here:
we have `const res = this._execute(worker, id)` so res is a raw Promise
and this raw Promise is returned by the function
if we use async await, we could also write `const res = await this._execute(worker, id)`
But maybe we don't want that, because we want to decide the user of our tool when he wants to await the response on its own

Reply:
I vote for removing the async, since it makes life so much easier without any drawbacks.

Reply:
Just thinking if removing the async can break compatibility for end-users mmmmmm
I guess not right? Then we can just remove it